front/app/containers/Admin/settings/policies/PageEditor.tsx

```diff
@@ -17,7 +17,7 @@ import { handleAddPageFiles, handleRemovePageFiles } from 'services/pageFiles';
 
```

Comment:
This is a component used to edit 'policies', which are special pages (Terms and Conditions and Privacy Policy). These are not editable from anywhere else. We put them in a seperate place from the other pages because you are not allowed to edit them to the same extent as other pages (e.g. you cannot change their slugs).